libs/ledger-live-common/src/families/hedera/bridge.integration.test.ts

```diff
@@ -93,7 +94,9 @@ const hedera: CurrenciesData<Transaction> = {
           }),
           expectedStatus: (account, _, status) => {
             return {
-              amount: account.balance.minus(status.estimatedFees),
+              amount: account.balance.minus(
+                status.estimatedFees.toNumber() * estimatedFeeSafetyRate
```

Comment:
we shouldn't have to convert back to number. instead it should be `status.estimatedFees.times(estimatedFeeSafetyRate)` (or multipliedBy)